application_form/models/application.py

```diff
@@ -79,9 +71,7 @@ class ApplicationApartment(models.Model):
     application = models.ForeignKey(
         Application, on_delete=models.CASCADE, related_name="application_apartments"
     )
-    apartment = models.ForeignKey(
-        Apartment, on_delete=models.CASCADE, related_name="application_apartments"
-    )
+    apartment_uuid = models.UUIDField(default=uuid4, verbose_name=_("apartment uuid"))
```

Comment:
This should probably not have a default value since this value is coming from another system. I would recommend settings the default to null if you want to be able to create these without first getting info from ES.

Reply:
That is true, I'll fix this.

Reply:
Changed default to None